simtbx/diffBragg/phil.py

```diff
@@ -542,6 +542,9 @@
   .help = if True, let the values of init.diffuse_sigma and init.diffuse_gamma
   .help = be used to define the diffuse scattering. Set e.g. fix.diffuse_sigma=True in order to refine them
   .expert_level = 10
+symmetrize_diffuse = False
```

Comment:
@dermen How about we make the default here true?

Reply:
I guess it has potential to increase the iteration time (maybe not as much as stenciling), which is why I left it False by default .. Happy to toggle it though

Reply:
Made default Tru in subsequent commit